frontend/src/pages/PhishingEmailInfo/utils.js

```diff
@@ -21,7 +21,7 @@ export const positiveMessages = actionType => {
     case 'email_deleted':
       return 'You deleted this email!';
     default:
-      return 'This was a high risk thing to do!';
+      return 'You deleted this email!';
```

Comment:
I don't think the default message should be email deleted. Make it the opposite of negative for now as we're going to review the content for these pages anyway.
```suggestion
      return 'This was a low risk thing to do!';
```